src/cpp/session/modules/SessionOverlay.R

```diff
@@ -12,3 +12,10 @@
 # AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
 #
 #
+
+.rs.addFunction("invokeServerRpc", function(method, ...)
+{
+   # callback to session to invoke server RPC
+   args <- list(...)
+   .Call("rs_invokeServerRpc", method, .rs.scalarListFromList(args))
```

Comment:
nit: use `PACKAGE = "(embedding)"` as otherwise `.Call()` will search all loaded DLLs for a routine called `rs_invokeServerRpc`, rather than just directly look for our own embedded routine.

Reply:
For sure, I can add that. I just copied this out of Pro into OS. Should we do a pass on the whole code base to make sure we're setting the package everywhere?

Reply:
I think that's reasonable, although until now I've just tried to make that change piecemeal whenever I encounter it in code that I'm modifying (or stumble upon for other reasons).